{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "1cbdca15_350796f4",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 1131
      },
      "writtenOn": "2022-12-05T15:37:40Z",
      "side": 1,
      "message": "recheck test_stable_device_rescue_disk_virtio unrelated failure",
      "revId": "a901a18d476927b2dfc6ac02dbca9ed53fe9edd3",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    }
  ]
}